appengine/express-memcached-session/server.js

```diff
@@ -24,29 +24,33 @@ var app = express();
 
 app.use(cookieParser());
 app.use(session({
-    secret: 'appengineFTW',
-    key: 'test',
-    proxy: 'true',
-    store: new MemcachedStore({
-        hosts: [process.env.MEMCACHE_URL || '127.0.0.1:11211']
-    })
+  secret: 'your-secret-here',
+  key: 'test',
+  proxy: 'true',
+  store: new MemcachedStore({
+    hosts: [process.env.MEMCACHE_URL || '127.0.0.1:11211']
+  })
 }));
 
 app.get('/', function(req, res){
-    publicIp.v4(function (err, ip) {
-
-        // This shows the IP for each 
-        res.write('<div>' + ip + '</div>');
-   
-        if(req.session.views) {
-            ++req.session.views;
-        } else {
-            req.session.views = 1;
-        }
-        res.end('Viewed <strong>' + req.session.views + '</strong> times.');
-    });
+  publicIp.v4(function (err, ip) {
+
+    // This shows the IP for each 
+    res.write('<div>' + ip + '</div>');
+
+    if(req.session.views) {
+      ++req.session.views;
+    } else {
+      req.session.views = 1;
+    }
+    res.end('Viewed <strong>' + req.session.views + '</strong> times.');
+  });
 });
 
-http.createServer(app).listen(process.env.PORT || 8080, function() {
+if (module === require.main) {
```

Comment:
why is this needed here? Is it for tests? I don't often see this in other samples.